lib/sqlalchemy/sql/selectable.py

```diff
@@ -4716,7 +4747,7 @@ def _get_display_froms(
                 f
                 for f in froms
                 if f
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325